meson.build

```
@@ -0,0 +1,201 @@
+project('vips', 'c', 'cpp',
+    version: '8.13.0',
+    meson_version: '>=0.56',
+    default_options: [
+        'c_std=c99'
+    ]
+)
+
+version_parts = meson.project_version().split('.')
+version_major = version_parts[0]
+version_minor = version_parts[1]
+version_patch = version_parts[2]
+
+# rules:
+# sources changed: increment revision
+# binary interface changed: increment current, reset revision to 0
+#   binary interface changes backwards compatible?: increment age
+#   binary interface changes not backwards compatible?: reset age to 0
+library_current = '57'
+library_revision = '0'
+library_age = '15'
+library_version = '.'.join ([library_current, library_revision, library_age])
```

I think this causes the soname to be generated incorrectly as `libvips{-cpp}.so.57` instead of `libvips{-cpp}.so.42`. Also, the `library_revision` and `library_age` should probably be reversed.
```suggestion
library_current = 57
library_age = 15
library_revision = 0
library_version = '@0@.@1@.@2@'.format(library_current - library_age, library_age, library_revision)
```

---

Oops, applied, thanks :)